query/src/sessions/session_mgr.rs

```diff
@@ -409,50 +409,6 @@ impl SessionManager {
         Ok(op)
     }
 
-    pub async fn reload_config(&self) -> Result<()> {
-        // TODO(xp): Potential race condition if fields are updated one by one.
-        //           These fields should all be got prepared then update in one atomic update.
-
-        let config = {
-            let mut config = self.conf.write();
```

Comment:
Maybe we can remove the lock of config if we won't reload it?

Reply:
Yup